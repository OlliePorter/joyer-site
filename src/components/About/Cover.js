import React from 'react';
import Typography from '@material-ui/core/Typography';
import Button from '@material-ui/core/Button';
import KeyboardArrowDownIcon from '@material-ui/icons/KeyboardArrowDown';
import { makeStyles } from '@material-ui/core/styles';
import theme from '../../theme';
import useMediaQuery from '@material-ui/core/useMediaQuery';

const useStyles = makeStyles({
    title: {
      color: theme.palette.primary.light,
      fontWeight: 300,
      margin: '20px',
      [theme.breakpoints.down('sm')]: {
        margin: '30px',
        paddingTop: '30px',
      },
    },
    subtitle: {
      color: theme.palette.primary.light,
      padding: '8px',
      fontWeight: 300,
      width: '60%',
      [theme.breakpoints.down('sm')]: {
        width: '80%',
      },
    },
    heroContent: {
      backgroundImage: 'url(/airship.png)',
      backgroundSize: 'cover',
      backgroundRepeat: 'no-repeat',
      backgroundPosition: 'center',
      minHeight: '45vh',
      [theme.breakpoints.down('sm')]: {
        minHeight: '80vh',
      },
    },
    heroOverlay: {
      backgroundColor: 'rgba(0, 0, 0, 0.5)',
      position: 'absolute',
      display: 'flex',
      flexDirection: 'column',
      alignItems: 'center',
      justifyContent: 'center',
      minHeight: '45vh',
      width: '100%',
      [theme.breakpoints.down('sm')]: {
        minHeight: '80vh',
      },
    },
    button: {
      width: '179px',
      color: 'white',
      backgroundColor: theme.palette.primary.main,
      height: '52px',
      fontSize: '17px',
      borderRadius: '50px',
      margin: '30px',
      textTransform: 'none',
      "&:hover": {
      background: theme.palette.primary.dark,
      },
    },
    downIcon: {
      color: 'white',
      transform: 'scale(1.5)',
      position: 'relative',
      top: '200px',
    },
});

export default function Cover() {
  const classes = useStyles();

    return (
    <>
    <div className={classes.heroContent}>
      <div className={classes.heroOverlay}>
        <Typography component="h2" variant="h3" align="center" className={classes.title}>
          Our Mission:
        </Typography>
        <Typography component="h2" variant="h6" align="center" className={classes.subtitle}>
        Promote collective wellbeing through stronger, healthier relationships.
        </Typography>
      </div>
    </div>
    </>
    );
}
