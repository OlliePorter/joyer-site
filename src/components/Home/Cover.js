import React from 'react';
import Typography from '@material-ui/core/Typography';
import { makeStyles } from '@material-ui/core/styles';
import theme from '../../theme';
import SpeechBubble from './SpeechBubble'

const useStyles = makeStyles({
    title: {
      color: theme.palette.primary.light,
      fontWeight: 300,
      margin: '20px',
      [theme.breakpoints.down('md')]: {
        margin: '30px',
        paddingTop: '15px',
      },
    },
    subtitle: {
      color: theme.palette.primary.light,
      padding: '8px',
      fontWeight: 300,
      width: '60%',
      [theme.breakpoints.down('sm')]: {
        width: '80%',
      },
    },
    heroContent: {
      backgroundImage: 'url(/joyer-group.png)',
      backgroundSize: 'cover',
      backgroundRepeat: 'no-repeat',
      backgroundPosition: 'center',
      minHeight: '80vh',
      [theme.breakpoints.down('sm')]: {
        minHeight: '95vh',
      },
    },
    heroOverlay: {
      backgroundColor: 'rgba(0, 0, 0, 0.5)',
      position: 'absolute',
      display: 'flex',
      flexDirection: 'column',
      alignItems: 'center',
      justifyContent: 'center',
      minHeight: '80vh',
      width: '100%',
      [theme.breakpoints.down('sm')]: {
        minHeight: '95vh',
      },
    },
});

export default function Cover() {
  const classes = useStyles();

    return (
    <>
      <div className={classes.heroContent}>
        <div className={classes.heroOverlay}>
          <Typography component="h2" variant="h3" align="center" className={classes.title}>
            The Personal CRM. Reimagined.
          </Typography>
          <Typography component="h2" variant="h6" align="center" className={classes.subtitle}>
            JOYER is a wellbeing app for building high quality social connections. Itâ€™s the best way to maintain strong relationships and bring balance to your headspace.
          </Typography>
          <SpeechBubble title={"'I feel so much closer to my friends after using this app.'"} imageLink={"'/deanna.jpeg'"}/>
          <SpeechBubble/>
          <SpeechBubble/>
        </div>
      </div>
    </>
    );
}
