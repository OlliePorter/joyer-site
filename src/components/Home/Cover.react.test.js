import React from 'react';
import Cover from './Cover';
import renderer from 'react-test-renderer';

it('Cover renders correctly', () => {
   const tree = renderer.create(
     <Cover />
   ).toJSON();
    expect(tree).toMatchSnapshot();
 });