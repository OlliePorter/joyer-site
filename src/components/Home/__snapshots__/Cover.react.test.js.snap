// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cover renders correctly 1`] = `
<div
  className="makeStyles-heroContent-3"
>
  <div
    className="makeStyles-heroOverlay-4"
  >
    <h2
      className="MuiTypography-root makeStyles-title-1 MuiTypography-h3 MuiTypography-alignCenter"
    >
      Love Your Clan.
    </h2>
    <h2
      className="MuiTypography-root makeStyles-subtitle-2 MuiTypography-h6 MuiTypography-alignCenter"
    >
      JOYER is a wellbeing app which cultivates relationships. It's the best way to maintain strong connections and bring balance to your social headspace.
    </h2>
    <button
      className="MuiButtonBase-root MuiButton-root MuiButton-contained makeStyles-button-5"
      disabled={false}
      onBlur={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="button"
    >
      <span
        className="MuiButton-label"
      >
        Coming Soon
      </span>
    </button>
    <svg
      aria-hidden="true"
      className="MuiSvgIcon-root makeStyles-downIcon-6"
      focusable="false"
      role="presentation"
      viewBox="0 0 24 24"
    >
      <path
        d="M7.41 8.59L12 13.17l4.59-4.58L18 10l-6 6-6-6 1.41-1.41z"
      />
    </svg>
  </div>
</div>
`;
